package.json

```diff
@@ -73,14 +73,15 @@
     "@typescript-eslint/eslint-plugin": "~5.27.1",
     "@typescript-eslint/parser": "~5.27.1",
     "copyfiles": "^2.4.1",
-    "eslint": "^8.17.0",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-prettier": "^4.2.1",
+    "eslint": "^7.32.0",
+    "eslint-config-prettier": "^6.15.0",
+    "eslint-plugin-import": "^2.23.4",
+    "eslint-plugin-prettier": "^3.4.0",
     "handlebars": "^4.7.7",
     "jest": "^26.6.3",
     "jest-junit": "^13.2.0",
     "npm-run-all": "^4.1.5",
+    "prettier": "^2.6.2",
```

Comment:
Added `prettier` reference back so that node version lower than 14 is supported without errors.